src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        int maxNumberBall = 100;
```

Comment:
```suggestion
    private final Random random = new Random();
    private static final int MAX_BOUND = 100;

    public Ball getRandomBall() {
```
still a local variable, constants are static and final, and we use CONSTANT_CASE for naming

Reply:
I hope everything is correct!